seed/challenges/01-front-end-development-certification/bootstrap.json

```diff
@@ -2144,7 +2144,8 @@
       "title": "Give Each Element a Unique ID",
       "description": [
         "We will also want to be able to use jQuery to target each button by its unique id.",
-        "Give each of your buttons a unique id like, starting with <code>target1</code> and ending with <code>target6</code>."
+        "Give each of your buttons a unique id like, starting with <code>target1</code> and ending with <code>target6</code>.",
```

Comment:
Not your fault, but there is a weird grammar error here:
> Give each of your buttons a unique id like, starting with . . . 

Can you please change to:

> Give each of your buttons a unique id, starting with . . .